src/components/feed/AddNode/AddNode.tsx

```diff
@@ -33,12 +35,24 @@ function getInitialState() {
 }
 
 const AddNode: React.FC<AddNodeProps> = ({
+  isCreator,
```

Comment:
nitpick: it would be better to use a variable name such as `isOwner`, or perhaps even better, `readOnly`.

The creator of a feed might not necessarily have permission to click the buttons, it is the feed owner who has those permissions, and feed ownership can be changed.

I think a variable name `readOnly` is more clear for what the purpose of the variable is, and it is general so it leaves room for future features. For instance, one day we might want to implement some toggle for "Editing/Reading" mode similar to Google Docs and Vim.